### PR TITLE
Point API in TOC to new API pages

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/ef/core/?branch=Rick-Anderson-patch-1)

### CODE REVIEW

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/Rick-Anderson-patch-1/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201710101942035104-516%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qa%2B%2FpzxpcS1Em0LXxW99vyUfTykmBTwB5T3Iiebvim4%3D&st=2017-10-10T19%3A39%3A26Z&se=2017-11-10T19%3A44%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/Rick-Anderson-patch-1/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201710101951577776-516%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jpOr%2FdE6iKTS4tpGZaR2JUO6q%2Fpfk8ymJwvaURiMk4U%3D&st=2017-10-10T19%3A49%3A47Z&se=2017-11-10T19%3A54%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/Rick-Anderson-patch-1/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201710101958454601-516%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=v8MUdIB7Xx%2F9iYgg0Sd%2Biwsy477JqOMzhCpJMPv5VSY%3D&st=2017-10-10T19%3A54%3A30Z&se=2017-11-10T19%3A59%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Rick-Anderson** [comment]
@divega  [Review URL](https://review.docs.microsoft.com/en-us/ef/core/?branch=Rick-Anderson-patch-1) You need to ^F5 to force cache update to see the TOC changes.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/Rick-Anderson-patch-1/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201710102006593630-516%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jEgNVGVtl7q03qQV9EV2YyjYionJkPEk8EoKWl74Ses%3D&st=2017-10-10T20%3A02%3A47Z&se=2017-11-10T20%3A07%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Rick-Anderson** [comment]
>There are additional links to API reference [here](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/index.md) 

These links
****
image

I could make API a node and add all the links under it. Why aren't those API's in https://docs.microsoft.com/en-us/dotnet/api/?view=efcore-2.0 ( or 1.0 or 1.1 ) ?

>I wonder what the right thing to do is with the fact that with this change we now have links in the ToC that point to pages that are outside of the scope of the EF docs. E.g. once the customer clicks on them, the ToC itself disappears and context is lost. Should we have some convention to decorate such links?

I'll investigate the docs.microsoft.com standard. Bread crumbs would nice - but I don't know if they're supported for this.

**Comment by Rick-Anderson** [comment]
@dend  We need to surface the API for the [ASP.NET Core docs](https://docs.microsoft.com/en-us/aspnet/core/) and the [EF Core docs](https://docs.microsoft.com/en-us/ef/core/get-started/). 

The [.NET docs](https://docs.microsoft.com/en-us/dotnet/framework/) uses the universal header to surface them - and that works well

image

That doesn't work for most APIs as they are burred in the untenable drop down.  See this [.NET API Browser Improvements](https://mseng.visualstudio.com/CSI/_workitems?id=1079600)

[ASP.NET Core docs](https://docs.microsoft.com/en-us/aspnet/core/)  API surface is not great and I'd like to be consistent:

image

Should I schedule a meeting so we can discuss short and mid term plans?

**Comment by dend** [comment]
@Rick-Anderson do you have any thoughts about how we can fit that in the UHF?

**Comment by Rick-Anderson** [comment]
Any chance we could pass the API (entity framework) as a query string to the UHF?

File: entity-framework/toc.md [hunk 1]
```diff
@@ -127,7 +125,9 @@
 ####[Upgrading from 1.0 RC1 to RC2](core/miscellaneous/rc1-rc2-upgrade.md)
 ####[Upgrading from 1.0 RC2 to RTM](core/miscellaneous/rc2-rtm-upgrade.md)
 ####[Upgrading to EF Core 2.0](core/miscellaneous/1x-2x-upgrade.md)
+###[Documentation](http://msdn.com/data/ef)
```
Comment (divega) [inline entity-framework/toc.md]: This line belongs under the EF6 section.

File: entity-framework/toc.md [hunk 2]
```diff
@@ -127,7 +125,9 @@
...
+<!-- API Reference -->
+##[API Reference](https://docs.microsoft.com/en-us/dotnet/api/?view=efcore-2.0)
 
 ##[Entity Framework 6](ef6/index.md)
-###[Documentation](http://msdn.com/data/ef)
-###[API Reference](https://msdn.microsoft.com/library/dn223258(v=vs.113).aspx)
+###[MSDN API Reference](https://msdn.microsoft.com/library/dn223258(v=vs.113).aspx)
```
Comment (divega) [inline entity-framework/toc.md]: Migration of EF6 conceptual topics is pending in https://github.com/aspnet/EntityFramework.Docs/pull/439. Do you know what happens with API docs afterwards?

**Comment by divega** [comment]
@Rick-Anderson @dend I have updated the links in the ToC and root index page in 64978e55b60b0f5544cabd8941387f13a1729ac6 and b5c562f6ac0555e94d0204bc27605afc1bb2838c. 

I am using ⤤ in these two places as a convention to signal links that will exit the context of the EF docs. It is not much worse than what we had before although it would be nice if there was something consistent and it wasn't necessary to leave the context of the EF docs.

I don't think this PR is necessary anymore.

**Comment by divega** [comment]
Closing as per my previous comment, since this doesn't seem necessary anymore.

**Pull request closed without merging**